packages/syft-cli/tests/version_test.py

```diff
@@ -0,0 +1,12 @@
+# third party
+from cli import app
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_version():
+    result = runner.invoke(app, ["version"])
+    assert result.exit_code == 0
+    assert "CLI" in result.stdout
+    assert "Library" in result.stdout
```

Comment:
MAYBE: When we modify syft version to be optional (during syft version), I think we would modify this line also (to optionally check Library string), when people run syft cli tests without installing syft .